.github/workflows/test.yml

```
@@ -0,0 +1,92 @@
+# Terraform Provider testing workflow.
+name: "Tests"
```

I think it might be nice to have a more specific name since we will use this as a required status check.
```suggestion
name: "Acceptance tests"
```

---

it also includes the build and generate scripts. i think i can specifically name the tests job though